Form/Step.php

```diff
@@ -28,6 +28,11 @@ class Step implements StepInterface {
 	protected $type = null;
 
 	/**
+	 * @var array
+	 */
+	protected $form_options = array();
```

Comment:
`$formOptions` please. The string for the step config should be `'form_options'` like you did, but not the property.